schema.json

```
@@ -57,15 +59,20 @@
                         "$ref": "#/definitions/entity"
                     }
                 ]
-            }
+            },
+            "minItems": 1,
+            "type": "array",
+            "uniqueItems": true
         },
         "date-released": {
-            "$ref": "#/definitions/date"
+            "$ref": "#/definitions/date",
+            "description": "The date the work has been released."
         },
         "doi": {
             "$ref": "#/definitions/doi"
         },
         "identifiers": {
+            "description": "The persistent identifiers of the software or dataset (for identifiers that are not DOIs).",
```

```suggestion
            "description": "The persistent identifiers of the software or dataset.",
```

This does take DOIs as well, so may be misleading to say *for identifiers that aren't DOIs*. Should we say something here about the relation to `doi`, i.e., "for cases where the software has more identifiers than a single DOI" or similar?

---

> Should we say something here about the relation to doi

Agree, this would be good

Additionally, maybe we should remove the word "persistent", since there could be URLs and other identifiers under this key

---

Yes:

```suggestion
            "description": "The identifiers of the software or dataset.",
```